config/esp32/components/chip/CMakeLists.txt

```diff
@@ -341,7 +346,10 @@ if((NOT CONFIG_USE_MINIMAL_MDNS) AND (CONFIG_ENABLE_WIFI_STATION OR CONFIG_ENABL
 endif()
 
 idf_component_get_property(main_lib main COMPONENT_LIB)
+idf_component_get_property(esp32_secure_cert_mgr_lib espressif__esp_secure_cert_mgr COMPONENT_LIB)
+
```

Comment:
Should this and the below line be added when `CONFIG_SEC_CERT_DAC_PROVIDER` is enabled?